### PR TITLE
Fix Issue with Container Actions Not Being Available on Windows Runners

### PR DESCRIPTION
**Pull request by AndrewKahr**
#### Changes

- Addresses issues with container registry found in #141
- Removed container registry for windows based builds and instead do everything locally on the runner
- Adjusted Job Names slightly to reflect base OS for clarity

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)

### CODE REVIEW

**Comment by AndrewKahr** [comment]
@webbertakken Let me know if this looks sufficient

File: .github/workflows/pull-requests.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
           digest: sha256:ef872b67450a680fbeebe1b037190b8f8fbfb04088531f422873915520e26e0f;
 
   build-ubuntu:
-    name: "🧪 Test build (${{ matrix.targetPlatform }})"
+    name: "🧪 Test Build Ubuntu (${{ matrix.targetPlatform }})"
```
Comment (webbertakken) [inline .github/workflows/pull-requests.yml:76]:
```suggestion
    name: "🧪 Test build Ubuntu (${{ matrix.targetPlatform }})"
```

File: .github/workflows/pull-requests.yml [hunk 2]
```diff
@@ -212,13 +212,8 @@ jobs:
         run: df -h
 
   build-windows:
-    name: "🧪 Test build (${{ matrix.targetPlatform }})"
+    name: "🧪 Test Build Windows (${{ matrix.targetPlatform }})"
```
Comment (webbertakken) [inline .github/workflows/pull-requests.yml:215]:
```suggestion
    name: "🧪 Test build Windows (${{ matrix.targetPlatform }})"
```

**Review by webbertakken — APPROVED** [review]
LGTM. Would you mind rebasing, so that this PR doesn't show 22 commits though?

**Comment by AndrewKahr** [comment]
@webbertakken I think it's rebased now? Let me know if there are any issues, this is my first time rebasing a PR.

**Comment by webbertakken** [comment]
Checking the workflow here https://github.com/game-ci/docker/pull/144

**Pull request merged**